### PR TITLE
removing view logs from about view and adding it to Logs table 

### PR DESCRIPTION
**Pull request by pavankalyan224847**
removed the click to vew logs link from about_tab.py file and added it to the schedule.tab

#1862

### Motivation and Context
click to view logs is supposed to be in the above table instead of about view so that users can check logs which have a return code of anything other than and check what went wrong 

### How Has This Been Tested?
i haven't tested it yet 

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ x] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist:

- [x ] I have read the [CONTRIBUTING](https://vorta.borgbase.com/contributing/) guide.
- [ x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

*I provide my contribution under the terms of the [license](./../LICENSE.txt) of this repository and I affirm the [Developer Certificate of Origin][dco].*

[dco]: https://developercertificate.org/

### CODE REVIEW

**Comment by pavankalyan224847** [comment]
ignore the second commit look in to the first one ie log link changed

**Comment by ThomasWaldmann** [comment]
You didn't change the .ui files:
- src/vorta/assets/UI/abouttab.ui
- .../scheduletab.ui

You can use qt designer to do that (which is part of qt creator).

**Comment by pavankalyan224847** [comment]
@ThomasWaldmann  now i am changing the ui files (I'll probably complete it by tomorrow) and can you tell me if the changes that i did in the schedule_tab.py and about_tab.py are proper or should i change those so that

**Comment by m3nu** [comment]
@pavankalyan224847 please don't ping any maintainer directly. Whoever has time will look into it. Looking at it now. Thanks!

**Comment by m3nu** [comment]
Right now this PR only has 2 changes to white space. Will wait for more substantial changes before reviewing. Thanks!

**Comment by pavankalyan224847** [comment]
ok i wont repeat it again but i have doubt can i use designer from qt5 tools because i am not able to use qt6 tools designer on ubuntu 22.04

**Comment by pavankalyan224847** [comment]
I have made a PR please look into it and let me know changes that have to be made

**Comment by ThomasWaldmann** [comment]
@pavankalyan224847 can you please in general first review all your changes (either locally `git diff` or on github "Files changed") yourself?

I see quite some changes here that are not related to the topic of this PR:
- some accidental changes by yourself
- some changes likely introduced by usage of qt designer
- some commits that do not have good content

You should fix such stuff first and produce a minimal and focussed PR, before review.

Maybe you want to use `git rebase -i master` to clean up your commits a bit. Be careful.

File: src/vorta/views/schedule_tab.py [hunk 1]
```diff
@@ -232,4 +232,8 @@ def populate_logs(self):
                 LogTableColumn.ReturnCode,
                 QTableWidgetItem(str(log_line.returncode)),
             )
+        self.logLink.setText(
+            f'<a href="file://{config.LOG_DIR}"><span style="text-decoration:'
+            'underline; color:#0984e3;">Click here</span></a> to view the logs.'
+        )
```
[the remark this answers is not in the document]
Comment (pavankalyan224847) [reply src/vorta/views/schedule_tab.py:238]: Shall I add this loginLink into the __init__ function which already exists  or should i create a new  __init__ function ?

[the remark this answers is not in the document]
Comment (ThomasWaldmann) [reply src/vorta/views/schedule_tab.py:238]: There can be only one `__init__` in that class and it already exists.

**Comment by pavankalyan224847** [comment]
yes I'll start working on it

**Comment by pavankalyan224847** [comment]
I am having trouble using git rebase can i just delete this PR  and create a new one ?

**Comment by real-yfprojects** [comment]
> I am having trouble using git rebase may i just delete this PR and create a new one ?

Instead you can also force push the new branch onto the old one. Also I recommend not to base PRs on the master branch of your fork.

**Comment by pavankalyan224847** [comment]
Yes I'll make sure that i'll make a different branch for pushing PRs and thank you for replying

**Pull request closed without merging**